docs/LanguageGuide.md

```diff
@@ -217,6 +219,23 @@ Sometimes, it's more convenient to bring a module's declarations into scope only
 (Vector2.update-x my-pos inc) ;; => (Vector2 11 20)
 ```
 
+### Enums / Algebraic types
+```clojure
+(deftype (Either a b)
+  (Left a)
```

Comment:
The members here must be wrapped in array, i.e. `(Left [a])`.

Reply:
True, forgot about that when I wrote that